boards/arm/nrf52833_pca10100/board.cmake

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(nrfjprog "--nrf-family=NRF52")
+board_runner_args(jlink "--device=nrf52" "--speed=4000")
+board_runner_args(pyocd "--target=nrf52833" "--frequency=4000000")
+include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
```

Comment:
that should probably be openocd, no pyocd, right @mbolivar ?

Reply:
or...

Reply:
probably, just adding the openocd here, too :)

Reply:
I added this myself :)

Reply:
I honestly don't know what firmware the interface MCU has on this board. I suppose if you added it it must work... ;)